external-crates/move/crates/move-bytecode-verifier/src/reference_safety/mod.rs

```diff
@@ -70,7 +70,7 @@ pub(crate) fn verify<'a>(
     resolver: &'a BinaryIndexedView<'a>,
     function_view: &FunctionView,
     name_def_map: &'a HashMap<IdentifierIndex, FunctionDefinitionIndex>,
-    meter: &mut impl Meter,
+    meter: &mut (impl Meter + ?Sized),
```

Comment:
Is it possible to change the declaration of `Meter` to something like `trait Meter: ?Sized` to reduce the number of changes here?

Reply:
Sadly not -- I tried that, but it is not allowed: https://play.rust-lang.org/?version=stable&mode=debug&edition=2021&gist=b43a8efd102d9c894f66a9293040d04f